src/components/DomainStatus.tsx

```diff
@@ -27,6 +30,13 @@ const withInlineError = (children: JSX.Element, type?: InlineMessageType) => (
   <InlineError type={type}>{children}</InlineError>
 );
 
+const humanizeEnsError = (error: ResolutionError, props: DomainStatusProps) => {
+  if (error.code === ResolutionErrorCode.RecordNotFound) {
```

Comment:
Are there any other error codes we need to catch and display for, because all the error messages will likely suck if one does.